solution/generate_planning_solutions.go

```diff
@@ -3,13 +3,14 @@ package solution
 import (
 	"container/heap"
 	"context"
+	"strconv"
+	"strings"
+
```

Comment:
remove space

Reply:
I think this space is intentional from go formatter to group different dependencies. The third party ones are grouped together.